### ISSUE TITLE
[rocblas] LLVMConfig.cmake missing

### ISSUE THREAD

**Issue by JorisBlanken** [rocm-arch/rocm-arch#164]
When building rocblas I'm getting:
```
CMake Error at /tmp/rocblas-6518267232824856842/rocblas/src/build/virtualenv/lib/python3.8/site-packages/Tensile/Source/lib/CMakeLists.txt:50 (find_package):
  Could not find a package configuration file provided by "LLVM" with any of
  the following names:

    LLVMConfig.cmake
    llvm-config.cmake

  Add the installation prefix of "LLVM" to CMAKE_PREFIX_PATH or set
  "LLVM_DIR" to a directory containing one of the above files.  If "LLVM"
  provides a separate development package or SDK, be sure it has been
  installed.
```

Cannot find LLVMConfig.cmake nor llvm-config.cmake on system, is there a missing dependency?

**Comment by tpkessler** [comment]
Hey @JorisBlanken, you're right. `rocblas` needs `llvm-amdgpu`. Could you please try the updated PKGBUILD from PR #158? I got it compiled in a clean chroot. Attention: You need a lot of RAM, around 50 - 60 GB, see https://wiki.archlinux.org/index.php/swap#Swap_file on how to create a swapfile.

**Comment by tpkessler** [comment]
Solved with #158
